File: codewar/Day-271/vowelCount.dart
import "dart:core";
// https://www.codewars.com/kata/54ff3102c1bad923760001f3/train/dart

void main() {
  //
  // List<String> myList = ["daniel", "hemmati", "shahin"];
  // String vowel = "aeiouAEIOU";
  // String test = 'abcde';
  // var listOfString = test.split('').where((element) => element.contains('a'));
  // print(listOfString);
  var exp = new RegExp(r"(\w+)");
  Iterable<RegExpMatch> matches = exp.allMatches("daniel");
  print(matches);
}

int getCount(String inputStr) {
  String vowel = "aeiou";
  var exp = new RegExp(r"[aeiou]", caseSensitive: false);
  return 9;
}

File: codewar/V1/Day-468/vowel_count.py
# https://www.codewars.com/kata/54ff3102c1bad923760001f3/train/python


def get_count(sentence):
    res = []
    for vowel in sentence:
        if vowel in "aeiou":
            res.append(vowel)

    return len(res)
    # by me 😏
    return len([vowel for vowel in sentence if vowel in "aeiou"])


a = "aeoiu"
print(get_count(a))
